libs/windy-sounding/types/client/persistentStorage.d.ts

```
@@ -0,0 +1,8 @@
+declare class PersistentStorage {
+  private readonly internalStorage;
+  constructor();
+  getItem(key: string): Promise<string | null>;
+  setItem(key: string, value: string): Promise<void>;
+}
+export declare const persistentStorage: PersistentStorage;
+export {};
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This export makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)